schemas/2014-04-01/Microsoft.Sql.json

```diff
@@ -105,10 +106,13 @@
         "name": {
           "oneOf": [
             {
-              "type": "string"
+              "type": "string",
```

Comment:
IIRC, I had to manually delete these `enum`s for resource names due to https://github.com/Azure/autorest/issues/2471 . Is this fixed?

Reply:
PR to fix this: https://github.com/Azure/autorest.azureresourceschema/pull/48